app/src/main/scala/org/alephium/explorer/service/TokenService.scala

```diff
@@ -284,4 +296,30 @@ object TokenService extends TokenService with StrictLogging {
       }
       .map(_ => ())
   }
+
+  private def getInOutAmount(address: Address, token: TokenId, from: TimeStamp, to: TimeStamp)(
+      implicit
+      ec: ExecutionContext,
+      dc: DatabaseConfig[PostgresProfile]
+  ): Future[(U256, U256, TimeStamp)] = {
+    run(
+      for {
+        in  <- sumAddressTokenInputs(address, token, from, to)
+        out <- sumAddressTokenOutputs(address, token, from, to)
+      } yield (in, out, to)
```

Comment:
indeed, it could have a small time interval where it could happen. I need to check if we could have `insertBlocks` as a `DBAction` rather than a `Future`, otherwise it  will be complicated to have a transactionally query

Reply:
mmm it's complicated to move to full `DBAction`, during the insert, if we miss the parent we need to recover that and fetch the missing parent from the node, so it's a Future.

Reply:
So this is a bug right? The possibility that the explorer could display `n` outputs for an `address` with `n - 1` input? Should this be addressed?

Reply:
:+1: I'll create an issue